core/templates/components/summary-tile/learner-story-summary-tile.component.ts

```diff
@@ -78,12 +101,23 @@ export class LearnerStorySummaryTileComponent implements OnInit {
     this.storyLink = this.getStoryLink();
     this.storyTitle = this.storySummary.getTitle();
     this.thumbnailBgColor = this.storySummary.getThumbnailBgColor();
+    let nextIncompleteNode = this.storySummary.getNodeTitles()[
+      this.completedNodeCount];
+    this.nextIncompleteNodeTitle = (
+      `Chapter ${this.completedNodeCount + 1}: ${nextIncompleteNode}`);
     if (!this.topicName) {
       this.topicName = this.storySummary.getTopicName();
     }
     this.starImageUrl = this.getStaticImageUrl('/learner_dashboard/star.svg');
   }
 
+  isDisplayAreaHome(): boolean {
```

Comment:
Do you only need this check for Home tab?

Reply:
Yes. It's because we display the next incomplete chapter in the story only on the Home tab.